go.mod

```diff
@@ -30,6 +30,8 @@ replace (
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 )
 
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
+
 replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.10.0
 
-replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
+go 1.13
```

Comment:
We should probably remove this if we're going with go 1.12 for the moment

Reply:
Fixed. Thanks!